src/main/java/Schmidt.java

```diff
@@ -0,0 +1,219 @@
+            System.out.println("\t" + (i + 1) + "." + tasks.get(i).toString());
+        }
+
+        System.out.println("------------------------------------------------------------");
+    }
+
+    /**
+     * This is a helper method to print the message after adding a task
+     * @param tasks The list of tasks
+     */
+    public static void printAddedTaskMessage(ArrayList<Task> tasks) {
+        System.out.println("------------------------------------------------------------\n" +
+                "Got it. I've added this task:\n" +
+                "\t" + tasks.get(tasks.size() - 1).toString() + "\n" +
+                " Now you have " + tasks.size() + " tasks in the list.\n" +
+                "------------------------------------------------------------");
+    }
+
+    /**
+     * This is a helper method to add a todo task
+     * @param tasks The list of tasks
+     * @param input The user input
+     */
+    public static void addTodoHelper(ArrayList<Task> tasks, String input) {
+        try {
+            String description = input.substring(5);
+            tasks.add(new Todo(description));
+        } catch (StringIndexOutOfBoundsException e) {
+            printValidCommands();
+        }
+
+        printAddedTaskMessage(tasks);
+    }
+
+    /**
+     * This is a helper method to add a deadline task
+     * @param tasks The list of tasks
+     * @param input The user input
+     */
+    public static void addDeadlineHelper(ArrayList<Task> tasks, String input) {
+        try {
+            String description = input.substring(9, input.indexOf("/by") - 1);
+            String by = input.substring(input.indexOf("/by") + 4);
+            tasks.add(new Deadline(description, by));
+        } catch (StringIndexOutOfBoundsException e) {
+            printValidCommands();
+        }
+
+        printAddedTaskMessage(tasks);
+    }
+
+    /**
+     * This is a helper method to add an event task
+     * @param tasks The list of tasks
+     * @param input The user input
+     */
+    public static void addEventHelper(ArrayList<Task> tasks, String input) {
+        try {
+            String description = input.substring(6, input.indexOf("/from") - 1);
+            String from = input.substring(input.indexOf("/from") + 6, input.indexOf("/to") - 1);
+            String to = input.substring(input.indexOf("/to") + 4);
+            tasks.add(new Event(description, from, to));
+        } catch (StringIndexOutOfBoundsException e) {
+            printValidCommands();
+        }
+
+        printAddedTaskMessage(tasks);
+    }
+
+    /**
+     * This is a helper method to mark or unmark a task as done
+     * @param tasks The list of tasks
+     * @param input The user input
+     * @param shouldMark A boolean to indicate if the task needs to be marked or unmarked as done
+     */
+    public static void markTaskHelper(ArrayList<Task> tasks, String input, boolean shouldMark) {
+        String[] tokens = input.split(" ");
+
```

Comment:
Can add some comments before each if to remind what kind of situation it handles now.